lib/kwalify_to_json_schema/limitations.rb

```diff
@@ -4,7 +4,6 @@ module Limitations
     DATE_TYPE_NOT_IMPLEMENTED = "Kwalify 'date' type is not supported and is ignored"
     TIME_TYPE_NOT_IMPLEMENTED = "Kwalify 'time' type is not supported and is ignored"
     TIMESTAMP_TYPE_NOT_IMPLEMENTED = "Kwalify 'timestamp' type is not supported and is ignored"
-    UNIQUE_NOT_SUPPORTED = "Kwalify 'unique' is not supported by JSON Schema and is ignored"
-    MAPPING_DEFAULT_VALUE_NOT_SUPPORTED = "Kwalify mapping default value is not supported by JSON Schema and is ignored"
+    UNIQUE_NOT_SUPPORTED = "Kwalify 'unique' within a mapping is not supported by JSON Schema and is ignored"
```

Comment:
I think we should be more specific

```suggestion
    UNIQUE_NOT_SUPPORTED_WITHIN_MAPPING = "Kwalify 'unique' within a mapping is not supported by JSON Schema and is ignored"
```